torch/_dynamo/eval_frame.py

```diff
@@ -884,6 +888,16 @@ def graph_with_interpreter(*args):
         graph,
     ).transform()
 
+    new_graph.meta["example_inputs"] = example_inputs
+    new_graph.meta["input_shape_constraints"] = constraints
+    new_graph.meta["inline_constraints"] = {
+        node.meta["val"].node.expr: var_to_range_map[node.meta["val"].node.expr]
+        for node in new_graph.graph.nodes
+        if node.op != "placeholder"
+        and "val" in node.meta
+        and re.match(r"^i\d+$", str(node.meta["val"]))
+    }
```

Comment:
@tugsbayasgalan FYI, to retrieve the inline constraints, we have to do this anyway, I decide to save it in the graph module so you can use it directly anywhere else.